lib/service_check.pm

```diff
@@ -45,6 +46,10 @@ our @EXPORT = qw(
 );
 
 our $hdd_base_version;
+our $support_ver_def  = '12+';
+our $support_ver_12   = '=12';
+our $support_ver_ge15 = '=15';
```

Comment:
should be >=15 or 15+ ?

Reply:
It should be '15+'. Updated code. Thanks.